apps/aecore/src/aec_peers_pool.erl

```diff
@@ -151,12 +151,12 @@
 %% The default probability of selecting a peer from the verified pool.
 -define(DEFAULT_SELECT_VERIFIED_PROB,    0.5).
 %% The default time without a peer being updated after which it get removed.
--define(DEFAULT_MAX_UPDATE_LAPSE,         30 * 24 * 50 * 60 * 1000). % 30 days
+-define(DEFAULT_MAX_UPDATE_LAPSE,         30 * 24 * 60 * 60 * 1000). % 30 days
```

Comment:
I see this was a typo but yet this fix is changing the default? Since this PR was explicitly intended not to change default values, shall this be moved to the subsequent PR that changes defaults?

Reply:
Ok, I will move it to the other PR.